device/arista/x86_64-arista_7800_sup/platform.json

```diff
@@ -1,7 +1,17 @@
 {
     "chassis": {
         "name": "DCS-7800-SUP1A",
-        "components": [],
+        "components": [
```

Comment:
@Staphylo just in case you missed this comment:
test_components were **skipped** because your API returns 0 number of components.
Would you want to make the API to return 3?

Refer: https://github.com/sonic-net/sonic-mgmt/blob/c05753cfe68313805c24fe4607645b528e17611c/tests/platform_tests/api/test_chassis.py#L274

Reply:
Ah you're right.
To avoid duplication in `show platform firmware status` we only report the components on the supervisor module and no longer on the chassis.
Please ignore my comment then.